spec/index.html

```diff
@@ -788,7 +789,7 @@ <h3>RDF Terms</h3>
       <span id="dfn-rdf-terms"><!-- obsolete term--></span><dfn class="export" data-lt="rdf term">RDF terms</dfn>.</p>
 
     <p>
-      [=IRIs=], [=literals=] and [=blank nodes=] are said to be <dfn class="export" data-lt-no-plural>atomic</dfn> [=RDF terms=].
+      [=IRIs=], [=literals=] and [=blank nodes=] are said to be <dfn data-lt="basic rdf term">basic [=RDF terms=]</dfn>.
```

Comment:
Not sure the `[=RDF terms=]` inside <dfn> wil work.

What I see in "preview" is "basic" gets a peramlink but the `[= =]` takes priority and is a link to "RDF term"


```suggestion
      [=IRIs=], [=literals=] and [=blank nodes=] are said to be <dfn data-lt="basic rdf term">basic RDF terms</dfn>.
```